package.json

```diff
@@ -3,11 +3,12 @@
   "version": "0.0.1",
   "private": true,
   "dependencies": {
-    "@material-ui/core": "latest",
+    "@material-ui/core": "^4.11.0",
     "@material-ui/icons": "latest",
     "@types/react": "latest",
     "@types/react-dom": "latest",
     "axios": "^0.20.0",
+    "npm-watch": "^0.7.0",
```

Comment:
I don't think we need this, it should be reloading on save anyway, and is for me locally.